### PR TITLE
Support colours

### PR DESCRIPTION
**Pull request by RRMoelker**
https://designsystem.amsterdam.nl/7awj1hc9f/p/63dbfc

### CODE REVIEW

**Comment by RRMoelker** [comment]
Suggest squash and commit

**Review by timvanoostrom — APPROVED** [review]

**Review by timvanoostrom — APPROVED** [review]

**Comment by timvanoostrom** [comment]
LGTM

**Comment by timvanoostrom** [comment]
Ah oh, no write access.

File: src/scss/_ams-colorpalette.scss [hunk 1]
```diff
@@ -26,7 +26,11 @@ $ams-form-field:          $ams-middengrijs;
 $ams-background-sidebars: #e5e5e5;
 $ams-top-menu:            #f3f3f3;
 
+$ams-support-valid:       #00A03C
```
Comment (janjaap) [inline src/scss/_ams-colorpalette.scss]: @RRMoelker Hex codes should be lowercase

**Review by aditudorache — APPROVED** [review]

**Pull request merged**